src/components/app/load-data.js

```diff
@@ -48,16 +38,18 @@ export const getInitialState = (
 export const loadData = (data, onLoadData) => {
   switch (data) {
     case 'random':
-      return formatSnapshots(getRandomHistory());
-    case 'mock':
-      return formatSnapshots(mockData);
+      return formatData(getRandomPipeline());
+    case 'lorem':
+      return formatData(mockData.lorem);
+    case 'animals':
+      return formatData(mockData.animals);
     case 'json':
```

Comment:
This feels odd having the random / lorem / animals in the loadData function i.e. mixing test / mock related items with loading real data. Could the config be changed whereby the env var is a path to the data? e.g.

```
DATA_DIR='./mock-data/animals.json' && npm start
```

As there are 2 different mental models with this e.g. real json is config based while the others are key based ("animals", "lorem")

Reply:
I think the issue is that there are 3 data loading methods: Direct import from a file, passing it as a prop, and async file request. This function/prop determines which of these to use and where to source it from. There are legitimate reasons why they can't all use the same source input method, as the 3 methods are used in different circumstances (i.e. testing, usage as a React component, and usage as a standalone app). The only reason I've added `lorem` and `animals` here is to allow one to run the test dataset in the browser, to make debugging tests easier.